### PR TITLE
DOCS-1953: Update freenove base dog tutorial for R2D2

### PR DESCRIPTION
**Pull request by sguequierre**
* Update language and screenshots
* Update instructions for local module local component flow

### CODE REVIEW

File: docs/tutorials/custom/custom-base-dog.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Control a Robot Dog with a Custom Viam Base Component"
+title: "CONTROL a Robot Dog with a Custom Viam Base Component"
```
Comment (npentrel) [inline docs/tutorials/custom/custom-base-dog.md:2]: LOL
```suggestion
title: "Control a Robot Dog with a Custom Viam Base Component"
```

File: docs/tutorials/custom/custom-base-dog.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ videos:
     "/tutorials/custom-base-dog/base-control-dog.webm",
     "/tutorials/custom-base-dog/base-control-dog.mp4",
   ]
-videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's Control tab on the Viam app open in a browser window."
+videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's CONTROL tab on the Viam app open in a browser window."
```
Comment (npentrel) [inline docs/tutorials/custom/custom-base-dog.md:13]:
```suggestion
videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's Control tab on the Viam app open in a browser window."
```
In alt text this is read as C O N T R O L which is annoying - so best not to change to all caps there

File: docs/tutorials/custom/custom-base-dog.md [hunk 3]
```diff
@@ -31,11 +31,11 @@ Viam natively supports a wheeled base model, but if you have a quadruped or othe
 This tutorial demonstrates how to add a custom base using [this robot dog kit and its open source code](https://github.com/Freenove/Freenove_Robot_Dog_Kit_for_Raspberry_Pi) as an example.
 
 <div class="alignleft">
-  {{<gif webm_src="/tutorials/custom-base-dog/base-control-dog.webm" mp4_src="/tutorials/custom-base-dog/base-control-dog.mp4" alt="A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's Control tab on the Viam app open in a browser window." max-width="400px">}}
+  {{<gif webm_src="/tutorials/custom-base-dog/base-control-dog.webm" mp4_src="/tutorials/custom-base-dog/base-control-dog.mp4" alt="A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's CONTROL tab on the Viam app open in a browser window." max-width="400px">}}
```
Comment (npentrel) [inline docs/tutorials/custom/custom-base-dog.md:34]:
```suggestion
  {{<gif webm_src="/tutorials/custom-base-dog/base-control-dog.webm" mp4_src="/tutorials/custom-base-dog/base-control-dog.mp4" alt="A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's control tab on the Viam app open in a browser window." max-width="400px">}}
```

**Review by npentrel — APPROVED** [review]

File: docs/tutorials/custom/custom-base-dog.md (hunk 1 above)
Comment (sguequierre) [reply docs/tutorials/custom/custom-base-dog.md:2]: oops .. when find replace goes wrong lol

**Pull request merged**